File: jules-scratch/verification/verify.py
from playwright.sync_api import sync_playwright

def run():
    with sync_playwright() as p:
        browser = p.chromium.launch(headless=True)
        page = browser.new_page()
        page.goto("http://localhost:8000/index.html")
        page.wait_for_selector("h1")
        page.screenshot(path="jules-scratch/verification/verification.png")
        browser.close()

run()

File: verify.js
const playwright = require('playwright');

async function main() {
  const browser = await playwright.chromium.launch();
  const page = await browser.newPage();
  await page.goto('http://localhost:8080');
  await page.screenshot({ path: 'screenshot.png' });
  await browser.close();
}

main();
